abis_mapping/templates/survey_metadata/mapping.py

```diff
@@ -520,7 +520,8 @@ def add_spatial_coverage(
         graph.add((geometry_node, utils.namespaces.GEO.asWKT, geometry.to_transformed_crs_rdf_literal()))
 
         if spatial_accuracy is not None:
-            graph.add((geometry_node, utils.namespaces.GEO.hasMetricSpatialAccuracy, spatial_accuracy))
+            accuracy = rdflib.Literal(spatial_accuracy, datatype=rdflib.XSD.double)
+            graph.add((geometry_node, utils.namespaces.GEO.hasMetricSpatialAccuracy, accuracy))
```

Comment:
looks good

Reply:
looks like the way to go when adding the accuracy to the graph.